volkr_munko/src/Css/FinanceStyled.js

```diff
@@ -0,0 +1,233 @@
+  }
+`;
+
+export const FinanceLeftInner = styled.div`
+  @media (max-width: 991px) {
+    margin: 0 20px;
+  }
+`;
+
+export const FinanceHeading = styled.div`
+  position: relative;
+  max-width: 565px;
+  width: 100%;
+  margin: 0 0 40px;
+  font-family: 'Quicksand-Bold';
+  font-size: 51px;
+  font-weight: bold;
+  line-height: 72px;
+  letter-spacing: -0.56px;
+  text-align: left;
+  text-transform: uppercase;
+  color: #14183e;
+  z-index: 2;
+
+  &::after {
+    content: '';
+    position: absolute;
+    top: -4px;
+    left: 0;
+    max-width: 427px;
+    width: 100%;
+    height: 68px;
+    z-index: -1;
+    background: linear-gradient(to bottom, #f8e9ed, #fbfbfb);
+  }
+
+  @media (max-width: 991px) {
+    width: 100%;
+    font-size: 24px;
+    font-weight: bold;
+    line-height: 31px;
+    letter-spacing: -0.3px;
+
+    &::after {
+      max-width: 288px;
+      width: 100%;
+      height: 46px;
+      top: -12px;
+    }
+  }
+`;
+
+export const FinanceDescription = styled.div`
+  max-width: 565px;
+  width: 100%;
+  font-family: 'Nunito-Regular';
+  font-size: 18px;
+  line-height: 31px;
+  letter-spacing: -0.18px;
+  color: #37415c;
+
+  @media (max-width: 991px) {
+    width: 100%;
+    font-size: 12px;
+    line-height: 18px;
+    letter-spacing: -0.12px;
+  }
+`;
+
+export const FinanceButton = styled.div`
+  position: relative;
+  width: 100%;
+  max-width: 210px;
+  height: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 12px;
+  margin: 50px 0 20px;
+  padding: 10px 20px;
+  font-family: 'Nunito-Bold';
+  font-size: 18px;
+  font-weight: 500;
+  letter-spacing: -0.18px;
+  text-align: left;
+  color: #000;
+  cursor: pointer;
+  background: radial-gradient(#fff, #e6e6e6);
+
+  &:hover {
+    background: radial-gradient(#767678, #3b3b3c);
+    color: #fff;
+  }
+
+  @media (max-width: 991px) {
+    margin: 46px 0 0 0;
+  }
+`;
+
+export const FinanceRightInner = styled.div`
+  @media (max-width: 991px) {
+    margin: 0 20px;
+  }
+`;
+
+export const FinanceRightMainHeading = styled.div`
+  max-width: 440px;
+  width: 100%;
+
+  @media (max-width: 1199px) {
+    max-width: 400px;
+    width: 100%;
+  }
+`;
+
+export const FinanceRightHeading = styled.div`
+  position: relative;
+  padding: 0 0 0 70px;
+  margin: 0 0 40px 0;
+  font-family: 'Quicksand-Bold';
+  font-size: 29px;
+  font-weight: bold;
+  line-height: 44px;
+  letter-spacing: -0.29px;
+  text-align: left;
+  color: #14183e;
+
+  &::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    top: 50%;
+    height: 2px;
+    max-width: 50px;
+    width: 100%;
+    background: #3b3b3c;
+  }
+
+  @media (max-width: 991px) {
+    font-size: 17px;
+    margin: 40px 0 20px;
+    padding: 0 0 0 50px;
+
+    &::before {
+      max-width: 30px;
+      width: 100%;
+    }
+  }
+`;
+
+export const FinanceImagePortion = styled.div`
+  padding: 40px;
+  border-radius: 23px;
+  border: solid 1px #cfcfcf;
```

Comment:
this is right but please make same as all like border :  1px solid color @aeshapatel025